addons/statemachine/fnc_clockwork.sqf

```diff
@@ -36,6 +35,7 @@ SCRIPT(clockwork);
 
     // When the list was iterated through, jump back to start and update it
     if (_tick >= count _list) then {
+        private _updateCode = _stateMachine getVariable QGVAR(updateCode);
```

Comment:
Can we be sure this variable doesn't change between function start and this, or between iterations?

Reply:
This is unscheduled afaik?
And there is nothing that executes user defined code between old L26 and the new L38.
So I'd call that safe.